DESCRIPTION

```diff
@@ -30,16 +32,19 @@ Imports:
     duckdb,
     fs,
     glue,
+    here,
     lubridate,
     parallelly,
-    purrr,
     readr,
+    rlang,
     sf,
     stats,
     stringr,
     tibble,
     xml2
 Suggests: 
+    furrr,
+    future,
```

Comment:
Really needed? Imported by `furrr`.

Reply:
OK see it is needed..

Reply:
needed for a helper... perhaps i could add the helper functions to `.Rbuildignore` and move them out of the package scope and let them live as just some scripts in a different folder.